mkdocs/docs/getting-started/quick-start.md

```diff
@@ -756,6 +948,122 @@ Iter8 introduces a Kubernetes CRD called Metric that makes it easy to use metric
+            value: |
+              (sum(increase(seldon_api_executor_server_requests_seconds_count{code!='200',seldon_deployment_id='$sid',kubernetes_namespace='$ns'}[${elapsedTime}s])) or on() vector(0)) / (sum(increase(seldon_api_executor_server_requests_seconds_count{seldon_deployment_id='$sid',kubernetes_namespace='$ns'}[${elapsedTime}s])) or on() vector(0))
+          provider: prometheus
+          sampleSize: iter8-seldon/request-count
+          type: Gauge
+          urlTemplate: http://seldon-core-analytics-prometheus-seldon.seldon-system/api/v1/query    
+        ---
+        apiVersion: iter8.tools/v2alpha2
+        kind: Metric
+        metadata:
+          name: mean-latency
+          namespace: iter8-seldon
+        spec:
+          description: Mean latency
+          jqExpression: .data.result[0].value[1] | tonumber
+          params:
+          - name: query
+            value: |
+              (sum(increase(seldon_api_executor_client_requests_seconds_sum{seldon_deployment_id='$sid',kubernetes_namespace='$ns'}[${elapsedTime}s])) or on() vector(0)) / (sum(increase(seldon_api_executor_client_requests_seconds_count{seldon_deployment_id='$sid',kubernetes_namespace='$ns'}[${elapsedTime}s])) or on() vector(0))
+          provider: prometheus
+          sampleSize: iter8-seldon/request-count
+          type: Gauge
+          units: milliseconds
+          urlTemplate: http://seldon-core-analytics-prometheus-seldon.seldon-system/api/v1/query      
+        ---
+        apiVersion: iter8.tools/v2alpha2
+        kind: Metric
+        metadata:
+          name: request-count
+          namespace: iter8-seldon
+        spec:
+          description: Number of requests
+          jqExpression: .data.result[0].value[1] | tonumber
+          params:
+          - name: query
+            value: |
+              sum(increase(seldon_api_executor_client_requests_seconds_sum{seldon_deployment_id='$sid',kubernetes_namespace='$ns'}[${elapsedTime}s])) or on() vector(0)
+          provider: prometheus
+          type: Counter
+          urlTemplate: http://seldon-core-analytics-prometheus-seldon.seldon-system/api/v1/query
+        ---
+        apiVersion: iter8.tools/v2alpha2
+        kind: Metric
+        metadata:
+          name: user-engagement
+          namespace: iter8-seldon
+        spec:
+          description: Number of feedback requests
+          jqExpression: .data.result[0].value[1] | tonumber
+          params:
+          - name: query
+            value: |
+              sum(increase(seldon_api_executor_server_requests_seconds_count{service='feedback',seldon_deployment_id='$sid',kubernetes_namespace='$ns'}[${elapsedTime}s])) or on() vector(0)
+          provider: prometheus
+          type: Gauge
+          urlTemplate: http://seldon-core-analytics-prometheus-seldon.seldon-system/api/v1/query
+        
+        ```
+
+
 
 ??? Note "Metrics in your environment"
     You can use metrics from any RESTful provider in Iter8 experiments. 
```

Comment:
Could  you please change this text  as follows?

`In this tutorial, the user-engagement metric is synthetically generated and served by mock New Relic service/Prometheus service`.

Reply:
Isn't this text shown across all the stacks? In the Seldon case i'm not using mock NewRelic.

Reply:
Ah ok - understand now. Yes will change.